### DISCUSSION TITLE
Create helm chart

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2995] [Feature Requests]
Hi.

I'm wondering if you have plans to create a helm chart. Will be much easier for all of us who use kubernetes and want to deploy this fast. 

I guess this question is for the community in general, maybe someone already working on this. 

I can contribute as well. 

Thx.

**Comment by benhaynes** [comment]
Good question — @WoLfulus ?

**Reply by Mahjouba91** [reply]
?

**Reply by WoLfulus** [reply]
Eventually, but not yet. I have different plans for K8s, but I'll have to look a bit deeper first.

**Reply by sapkra** [reply]
@WoLfulus Can you tease what you are thinking about?

**Reply by WoLfulus** [reply]
I want to check and test on my own sandbox projects first if it makes sense to write an actual operator for Directus and land something close to what we had previously on v8 regarding multi project setups, even though not as simple as it was.

I'll land a helm chart for sure, even if the operator isn't something we'll do, just not the time yet :)

**Reply by peteruithoven** [reply]
Any progress on this?

**Reply by botzill** [reply]
@peteruithoven I did push the chart I'm using now, you can use it from here: https://github.com/botzill/directus-helm

**Comment by rbbl-dev** [comment]
does it need to have an operator or would you accept a PR containing a chart without operator?

**Reply by rijkvanzanten** [reply]
@wrynegade?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
